ZombieSharp/Plugin/Listeners/ListenersExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace ZombieSharp.Plugin.Listeners;
+
+using CounterStrikeSharp.API.Core;
+
+public static class ListenersExtensions
+{
+    // how can we get all instances of ListenerBase?
+    // reflection/source generator should be used
+    // TODO: implement reflection/source generator
+    private static readonly List<ListenerBase> _listeners = [
```

Comment:
I gonna create separate project with source gen to analyze all Listeners in code to remove manual resolving